packages/components/Button/src/ButtonTokens.ios.ts

```diff
@@ -10,75 +10,72 @@ export const defaultButtonTokens: TokenSettings<ButtonTokens, Theme> = () =>
       width: '100%',
     },
     medium: {
-      padding: globalTokens.size80,
+      paddingHorizontal: globalTokens.size120,
       borderWidth: globalTokens.stroke.width10,
-      iconSize: 16,
+      borderRadius: globalTokens.corner.radius80,
+      minHeight: 40,
+      iconSize: 20,
       focused: {
         borderWidth: 0,
-        padding: globalTokens.size100,
       },
       hasContent: {
         minWidth: 96,
-        paddingHorizontal: globalTokens.size160 - globalTokens.stroke.width10,
         hasIconAfter: {
-          spacingIconContentAfter: globalTokens.size100,
+          spacingIconContentAfter: globalTokens.size80,
         },
         hasIconBefore: {
-          spacingIconContentBefore: globalTokens.size100,
+          spacingIconContentBefore: globalTokens.size80,
         },
         focused: {
-          paddingHorizontal: globalTokens.size160,
+          paddingHorizontal: globalTokens.size120,
         },
       },
     },
     small: {
-      padding: globalTokens.size80 - globalTokens.stroke.width10,
+      paddingHorizontal: globalTokens.size60,
       borderWidth: globalTokens.stroke.width10,
```

Comment:
Is the width of the button what you expect to see if paddingHorizontal and borderWidth are set? borderWidth might be adding an additional pixel that you're not expecting.

Reply:
the size looks good in the inspector